src/btree/bt_random.c

```diff
@@ -357,15 +389,17 @@ __wt_random_descent(WT_SESSION_IMPL *session, WT_REF **refp, uint32_t flags, WT_
     WT_PAGE *page;
     WT_PAGE_INDEX *pindex;
     WT_REF *current, *descent;
-    uint32_t i, entries, retry;
+    uint32_t i, entries;
     uint8_t descent_state;
-    bool eviction;
+    int retry;
+    bool eviction, search_inmem_page;
 
     *refp = NULL;
 
     btree = S2BT(session);
     current = NULL;
     retry = 100;
+    search_inmem_page = false;
```

Comment:
Rename this to sample_inmem_page?

Reply:
Update in [5119e77](https://github.com/wiredtiger/wiredtiger/pull/10437/commits/5119e771e9830dd9683687cdb6120cbdfc7f6fc1)